pos_edit_order_line/readme/CONFIGURE.rst

```diff
@@ -0,0 +1 @@
+Select PoS > configuration > enable flag "Allow Edit Order Line"
```

Comment:
To improve usability when there are more than 3 order lines, we suggest activating the "Large scrollbars" POS setting.